deno_dist/utils/jwt/jwt.ts

```diff
@@ -32,8 +32,7 @@ export function isTokenHeader(obj: any): obj is TokenHeader {
     obj !== null &&
     'alg' in obj &&
     Object.values(AlgorithmTypes).includes(obj.alg) &&
-    'typ' in obj &&
-    obj.typ === 'JWT'
+    (!('typ' in obj) || obj.typ === 'JWT')
```

Comment:
typ is case-insensitive, but left uppercase because uppercase is recommended for compatibility.

> While media type names are not case sensitive, it is RECOMMENDED that "JWT" always be spelled using uppercase characters for compatibility with legacy implementations.